src/framework/components/camera/component.js

```diff
@@ -144,6 +146,20 @@ class CameraComponent extends Component {
      */
     onPostRender = null;
 
+    /**
+     * A counter of requests of depth map rendering.
+     *
+     * @type {number}
```

Comment:
Does this need a @private?

Reply:
well spotted, thanks, I keep forgetting anything documented is public by default now. Done.